docs/api/supervisor/endpoints.md

```diff
@@ -1772,35 +1772,56 @@ Update Home Assistant OS
 | key      | type       | description                                      |
 | -------- | ---------- | ------------------------------------------------ |
 | unsupported | list | A list of reasons why a installation is marked as unsupported (container, dbus, docker_configuration, docker_version, lxc, network_manager, os, privileged, systemd) |
-| issues | list | A list of issues |
-| suggestions | list | A list of suggested actions |
+| issues | list | A list of [Issue models](api/supervisor/models.md#issues) |
+| suggestions | list | A list of [Suggestion models](api/supervisor/models.md#suggestion) actions |
 
 **Example response:**
 
 ```json
 {
   "unsupported": ["os"],
-  "issues": ["free_space"],
-  "suggestions": ["clear_snapshots"]
+  "issues": [
+    {
+      "uuid": "id",
```

Comment:
We should add a fake uuid here so the example response reflects what someone will get in return